components/awards/ScoreBasedAwards.js

```diff
@@ -19,11 +20,15 @@ import styles from '@styles/pages/Awards.module.scss'; // Page styles
 import { formatPerformances } from '@utils/performances'; // Format performances util
 import AwardPill from '@components/awards/FinalizePill.js'; // Award Finalize Pill
 
+import useSnackbar from '@utils/useSnackbar'; // Snackbar
+
 const PAGE_SIZE = 20; // Rows per page
 
 // Page: Performances
 export default function Performances({ award }) {
+  const { snackbarError } = useSnackbar();
   const [event] = Event.useContainer();
+  const [session] = useSession();
```

Comment:
I added session into this file in my last page, so could this be updated?

Reply:
I just rebased tonight. Was it merged recently? Which commit was it?

Reply:
https://github.com/uwblueprint/dancefest-web/commit/728deef9614aed2664cd4faa70ac4f4439fc2b1c

Reply:
Looks like there's also a conflict because of that change

Reply:
Ah okay, I'll resolve that. Thanks!